nmpc_examples/pipeline/control_input_data.json

```diff
@@ -0,0 +1 @@
+{"fs.pipeline.control_volume.flow_mass[*,1.0]": [553003.1393240632, 535472.4292275416], "fs.pipeline.control_volume.pressure[*,0]": [58.905880452751845, 58.92036532907178]}
```

Comment:
Where does this come from? I never like having a "data file" in a repository without the script to generate it. You can see the `get_control_inputs` function in my PR, which creates a `TimeSeriesData` object with this info. This can be serialized if necessary with `json.dump` and `TimeSeriesData.to_serializable`.

Reply:
These control inputs come from the results of `run_pipeline_nmpc.py`.
Yes, I think using `TimeSeriesData` to wrap these control inputs is more appropriate and easier to understand for users.